applications/processor/twitter-trend-processor/README.adoc

```diff
@@ -25,20 +25,6 @@ Response is an array of `locations` that encode the location's WOEID and some ot
 
 == Options
 
-//tag::configuration-properties[]
-Properties grouped by prefix:
-
-
-=== twitter.trend.closest
-
-$$lat$$:: $$If provided with a long parameter the available trend locations will be sorted by distance, nearest to the furthest, to the co-ordinate pair. The valid ranges for longitude is -180.0 to +180.0 (West is negative, East is positive) inclusive.$$ *($$Expression$$, default: `$$<none>$$`)*
-$$lon$$:: $$If provided with a lat parameter the available trend locations will be sorted by distance, nearest to the furthest, to the co-ordinate pair. The valid ranges for longitude is -180.0 to +180.0 (West is negative, East is positive) inclusive.$$ *($$Expression$$, default: `$$<none>$$`)*
-
-=== twitter.trend
-
-$$location-id$$:: $$The Yahoo! Where On Earth ID of the location to return trending information for. Global information is available by using 1 as the WOEID.$$ *($$Expression$$, default: `$$<none>$$`)*
-$$trend-query-type$$:: $$<documentation missing>$$ *($$TrendQueryType$$, default: `$$<none>$$`, possible values: `trend`,`trendLocation`)*
-//end::configuration-properties[]
+https://github.com/spring-cloud/spring-functions-catalog/tree/main/function/spring-twitter-function#12-configuration-options[See function catalog for configuration options]
```

Comment:
The auto generated link impl has limitations and relies on naming conventions. So when you see hardcoded links in here - that is the reason.